apps/sms/views/conversation/js/information.js

```diff
@@ -339,6 +346,42 @@ var VIEWS = {
           l10nContainsDateSetup(this.receivedTimestamp, message.timestamp);
           l10nContainsDateSetup(this.sentTimestamp, message.sentTimestamp);
           setL10nAttributes(this.sentTitle, 'message-sent');
+
+          var units = {
+            months: MONTH,
+            days: DAY,
+            hours: HOUR,
+            minutes: MINUTE,
+            seconds: SECOND
+          };
+          var lateness = message.timestamp - message.sentTimestamp;
+
+          this.latenessBlock.classList.remove('hide');
+          if (
+            !message.timestamp || 
+            !message.sentTimestamp || 
+            lateness < LATENESS_DISPLAY_LOWER_LIMIT
+          ) {
```

Comment:
nit: this is not very readable; maybe extract the condition to a carefully named boolean variable ? `shouldDisplayLatenessIndicator` for example :)